src/api/forecastAPI.ts

```diff
@@ -29,7 +29,7 @@ export interface ForecastResponse {
 }
 
 export async function getNoonForecasts(stationCodes: number[]): Promise<Forecast[]> {
-  const url = '/noon_forecasts/'
```

Comment:
so I addressed this issue in my pr by changing `REACT_APP_API_BASE_URL` to `http://localhost:8080/api` in `.env`. Can you change them back?

Reply:
will do

Reply:
My apologies for not being careful!

Reply:
no worries! it's all good :)